src/components/EloOverTime.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="col">
+    <div :id="ruleset" style="min-width: 310px; height: 400px;"></div>
+  </div>
+</template>
```

Comment:
Why not just toss it into `Leaderboard.vue` as well?

Reply:
I don't know how vue works.
I figured that `Leaderboard.vue` is for the leaderboard graph, and `EloOverTime` is for the elo over time graphs. I wasn't sure how much would be the same/different